var/spack/repos/builtin/packages/py-pyarrow/package.py

```diff
@@ -36,7 +38,7 @@ class PyPyarrow(PythonPackage, CudaPackage):
     depends_on('py-futures', type=('build', 'run'), when='@0.15.0:^python@:3.1.99')
```

Comment:
Can you update the deps? The numpy version required has changed, six is no longer a dep, and Python 3.6+ is now required.